src/main/java/seedu/address/logic/Messages.java

```diff
@@ -18,7 +18,7 @@ public class Messages {
             + "Friends: add-friend, edit-friend, delete-friend, find-friend, list-friend\n"
             + "Plans: add-plan, edit-plan, delete-plan, complete-plan, uncomplete-plan, find-plan, list-plan\n"
             + "Others: clear, help, exit\n";
-    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command. \n%1$s";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
```

Comment:
changing this will change all other output messages that uses this `MESSAGE_INVALID_COMMAND_FORMAT`

in add-plan, victor used `MESSAGE_MISSING_ARGUMENTS` instead